src/app/public/src/modules/sidebar/sidebar.component.ts

```diff
@@ -8,38 +8,39 @@ import { StacheRouteService } from '../shared';
   templateUrl: './sidebar.component.html',
   styleUrls: ['./sidebar.component.scss']
 })
-export class StacheSidebarComponent implements StacheNav, OnInit {
+export class StacheSidebarComponent implements StacheNav {
   @Input()
-  public routes: StacheNavLink[];
+  public set routes(value: StacheNavLink[]) {
+    this._routes = value ? value : this.routeService.getActiveRoutes();
+    this.childRoutes = this.filterRoutes(this._routes);
+  }
+
+  public get routes(): StacheNavLink[] {
+    return this._routes || [];
```

Comment:
Current tests never hit this "else" case.

Reply:
I think with the current `set` rules, It will never return the [] assignment in the get.  Likely unnecessary.